### PR TITLE
Nordpool doc update

### PR DESCRIPTION
**Pull request by AJediIAm**
## Type of change

- [X] Spelling, grammar or other readability improvements (`current` branch).
- [X] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **Documentation**
	- Enhanced clarity and detail in the Nord Pool integration documentation.
	- Updated configuration section to specify single configuration entry and base currency as EUR.
	- Restructured sensors section into a table format, detailing main sensors and adding peak/off-peak sensors.
	- Reformatted diagnostic sensors into a table for better presentation.
	- Expanded examples section with a template sensor for adding VAT and fixed costs, including detailed instructions.

### CODE REVIEW

**Comment by AJediIAm** [comment]
Apologies for making a bit of a mess out of this pull request. 
Your patience and guidance is appreciated.

File: source/_integrations/nordpool.markdown [hunk 1]
```diff
@@ -39,58 +39,92 @@ Currency:
...
           {{ ((cost + add_cost) * 1.25) | round(2, default=0) }}
 ```
 
 {% endraw %}
+
+### Energy Dashboard
+
+To use the Nordpool integration in the Energy dashboard, use the `Use an entity with current price` option when configuring grid consumption and production.
+
+<p class='img'>
```
Comment (gjohansson-ST) [inline source/_integrations/nordpool.markdown:128]: I think we should use markdown to add images https://www.markdownguide.org/basic-syntax/#images

**Review by gjohansson-ST — DISMISSED** [review]
Thanks for providing an update to this. Looks much better.
A small note on images and then I think it's very nice to go 👍

File: source/_integrations/nordpool.markdown [hunk 2]
```diff
@@ -39,58 +39,92 @@ Currency:
...
-### Diagnostic sensors
+A template sensor to add VAT and fixed cost is useful to get the actual energy cost in the energy dashboard. 
+
+### UI Template
 
-- Last updated - indicates when the market price was last updated.
-- Currency - The selected currency.
-- Exchange rate - EUR is the base currency so will show the exchange rate used on the market place.
+Create a helper using the UI.
+1. Go to `Settings`, `Devices & Services` and choose the `Helpers` tab at the top.
```
Comment (c0ffeeca7) [inline source/_integrations/nordpool.markdown]:
```suggestion
1. Go to {% my integrations title="**Settings** > **Devices & Services**" %} and at the top, choose the **Helpers** tab.
```

File: source/_integrations/nordpool.markdown [hunk 3]
```diff
@@ -39,58 +39,92 @@ Currency:
...
+2. Select `Create helper` at the bottom right.
```
Comment (c0ffeeca7) [inline source/_integrations/nordpool.markdown]:
```suggestion
2. In the bottom right corner, select **Create helper**.
```
we use bold to markup UI strings.

File: source/_integrations/nordpool.markdown [hunk 4]
```diff
@@ -39,58 +39,92 @@ Currency:
...
+3. Choose `Template` and `Template a sensor`.
```
Comment (c0ffeeca7) [inline source/_integrations/nordpool.markdown]:
```suggestion
3. Select **Template** and **Template a sensor**.
```

File: source/_integrations/nordpool.markdown [hunk 5]
```diff
@@ -39,58 +39,92 @@ Currency:
...
-          # Add fixed cost to the spot price and add VAT (25%)
+          # Add cost and additional fixed cost. Add VAT (25%) by multiplying with 1.25 and round to 2 digits: 
           {{ ((cost + add_cost) * 1.25) | round(2, default=0) }}
 ```
 
 {% endraw %}
+
+### Energy Dashboard
+
+To use the Nordpool integration in the Energy dashboard, use the `Use an entity with current price` option when configuring grid consumption and production.
```
Comment (c0ffeeca7) [inline source/_integrations/nordpool.markdown]:
```suggestion
To use the Nordpool integration in the **Energy** dashboard, when configuring grid consumption and production, use the **Use an entity with current price** option.
```

File: source/_integrations/nordpool.markdown (hunk 1 above)
Comment (c0ffeeca7) [reply source/_integrations/nordpool.markdown:128]: I think it's ok. I've seen html tags being used for images in other places.

Comment (gjohansson-ST) [reply source/_integrations/nordpool.markdown:128]: In that case. If you're happy then approve and merge it 👍

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @AJediIAm and @gjohansson-ST :+1:

**Pull request merged**